DashKit/Mocks/MockPodCommManager.swift

```diff
@@ -25,7 +25,11 @@ public class MockPodCommManager: PodCommManagerProtocol {
     var lastBasalProgram: BasalProgram?
     var lastTempBasal: TempBasal?
 
-    var podStatus: MockPodStatus
+    public var podStatus: MockPodStatus {
+        didSet {
+            dashPumpManager?.getPodStatus(completion: { _ in })
```

Comment:
Can you explain what's going on here? I don't understand this.

Reply:
The intent here is to report the `podStatus` when it is set.  

There are comments below:
```
// We can't call PodCommManagerDelegate methods on DashPumpManager because we do not have a real PodCommManager.
// We can use a direct reference to call the mirrored delegate methods, that expect PodCommManagerProtocol.
```

Based on those, it is calling the `PodCommManagerProtocol` `func getPodStatus(userInitiated: Bool, completion: @escaping (PodSDK.PodCommResult<PodStatus>) -> ())`. 

Did I misunderstand the comments?